File: 3858/3858.cs
public class Solution {
    public int MinimumBitwiseOrFromGrid(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MinimumBitwiseOrFromGrid(root.left), MinimumBitwiseOrFromGrid(root.right));
    }
}

File: 3858/3858.py
class Solution:
    def minimum_bitwise_or_from_grid(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        return 1 + max(self.minimum_bitwise_or_from_grid(root.left), self.minimum_bitwise_or_from_grid(root.right))